### ISSUE TITLE
Train the system using sample documents for improved accuracy

### ISSUE THREAD

**Issue by jillcfoley1** [GSA-TTS/document-extractor-poc#105]
As a user, in order to reduce my burden of editing the automatically extracted fields, I want accuracy to be improved through training.

## Preconditions

- [x] We are not happy with the accuracy and quality of the text extraction after story #72, which adds queries.

## Acceptance Criteria

- [x] The text extraction system is trained.
- [ ] Accuracy metrics are reviewed to determine whether the tool meets or exceeds baseline performance thresholds.

## Tasks

- [x] Gather/create 10 examples of each supported document type (e.g., W-2, 1099, DD214). (NOTE: The dataset will be split into 50% for training and 50% for testing. The training set is used to fine-tune or reinforce the extractor’s ability to recognize and extract key fields. The testing set is used to measure overall accuracy of extracted fields.)
  - [x] W2.
  - [x] DD214.
  - [x] 1099-NEC.
- [x] Create adapter(s) for W2. - @halprin 
- [x] Create adapter for DD214. - @halprin 
- [x] Create adapter for 1099-NEC.
- [x] Use adapters in code. - @saquino0827
